# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/revprog/hotfix-cstdint.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

Include <cstdint> for the fixed width integer types.

--- rvlprog-0.92.orig/include/protocol.hh	2022-02-06 16:20:22.000000000 +0100
+++ rvlprog-0.92/include/protocol.hh	2026-09-26 16:30:48.188183697 +0200
@@ -1,6 +1,7 @@
 #ifndef protocol_hh
 #define protocol_hh
 
+#include <cstdint>
 #include <string>
 #include <map>
 #include <vector>
